django_simple_api/decorators.py

```diff
@@ -13,6 +14,7 @@
     GenericType = (type(List[str]),)
 
 from django.views import View
+
```

Comment:
When importing third-party libraries, there should be no blank lines in the middle.